src/Umbraco.Forms.Integrations.Crm.ActiveCampaign/Client/src/property-editor/contact-mapping/contact-mapping-property-editor.element.ts

```diff
@@ -0,0 +1,198 @@
+
+  isDisabled(){
+    return !this.selectedContactField || !this.selectedFormField;
+  }
+
+  #onDeleteClick(idx: number){
+    this.contactMappingArray.splice(idx, 1);
+
+    this.value = JSON.stringify(this.contactMappingArray);
+    this.requestUpdate();
+    this.dispatchEvent(new CustomEvent('property-value-change'));
+  }
+
+  render() {
+    return html`
+      <div>
+        <uui-select
+          placeholder=${this.localize.term("formProviderWorkflows_SelectContactField")}
+          @change=${(e : UUISelectEvent) => this.#onContactSelectChange(e)}
+          .options=${
+            this.contactFields?.map((ft) => ({
+              name: ft.displayName,
+              value: ft.name,
+              selected: ft.name === this.selectedContactField,
+            })) ?? []}></uui-select>
+        <uui-select
+          placeholder=${this.localize.term("formProviderWorkflows_SelectFormField")}
+          @change=${(e : UUISelectEvent) => this.#onFormFieldSelectChange(e)}
+          .options=${
+            this.formdFields?.map((ft) => ({
+              name: ft.caption,
+              value: ft.id,
+              selected: ft.id === this.selectedFormField,
+            })) ?? []}></uui-select>
+      </div>
+
+      <div class="activecampaign-wf-required">
+        Mandatory fields: ${this.contactFields?.filter(c => c.required).map(c => c.displayName).join(", ")}
+      </div>
+
+      <div class="activecampaign-wf-button">
+        <uui-button look="primary" ?disabled=${this.isDisabled()} label=${this.localize.term("formProviderWorkflows_AddMapping")} @click=${this.#addButtonClick}></uui-button>
+      </div>
+
+      <div class="activecampaign-wf-table">
+        ${this.contactMappingArray.length > 0 
+          ? html`
+          <table>
+            <thead>
+              <tr>
+                <th>${this.localize.term("formProviderWorkflows_ContactField")}</th>
+                <th>${this.localize.term("formProviderWorkflows_FormField")}</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              ${map(this.contactMappingArray, (mapping, idx) => html`
+                <tr>
+                  <td>${mapping.contactField}</td>
+                  <td>${mapping.formField?.value}</td>
+                  <td>
+                    <uui-button
+                      label=${this.localize.term("formProviderWorkflows_delete")}
+                      look="secondary"
+                      color="default"
+                      @click=${() => this.#onDeleteClick(idx)}>
+                      <uui-icon name="delete"></uui-icon>
+                    </uui-button>
+                  </td>
+                </tr>
+              `)}
+            </tbody>
+          </table>
+          ` 
+          : html``}
+      </div>
+    `;
+  }
+
+  static styles = [
+    css`
+        .activecampaign-wf-required{
+          margin-top: 10px;
+          padding: 10px 15px;
+          background-color: #3333ff;
```

Comment:
This background color may not match the color we're using in previous version, so this should be changed.